### DISCUSSION TITLE
Documentation Mirrored to Wiki?

### DISCUSSION THREAD

**Discussion by NaluriMurai** [YSCEDC/YSCE#1] [General]
Since @decaff42 started working on creating documentation on YSFlight's subsystems, I'd like to suggest we mirror them to the wiki.(namely because it would be important to at least retain them as part of source so if it's forked and worked on offline, people can review those documents if need be)

I'd suggest we make a page dedicated to subjects relating to simulation implementation relating to physics, aerodynamics and other complex matters that aren't explicitly related to game logic or code in general.

**Comment by decaff42** [comment]
happy to mirror them or just exclusively have them in the Wiki, but not sure how we would structure the wiki for that.

**Reply by NaluriMurai** [reply]
That is actually a good question to be honest - we might want to figure that out.
